docs/Monitored_Only_ZFS.md

```diff
@@ -133,35 +107,48 @@ Note that VM Disks (ata-VBOX_HARDDISK...) will be mapped as /dev/sd devices. Use
     vagrant ssh oss1
     sudo -s
     zpool create oss1 -o cachefile=none raidz2 /dev/disk/by-id/ata-VBOX_HARDDISK_VB6f41df02-0d5d2a15 /dev/disk/by-id/ata-VBOX_HARDDISK_VB06b563a9-2539af7b  /dev/disk/by-id/ata-VBOX_HARDDISK_VBb2ec79fb-b900b724 /dev/disk/by-id/ata-VBOX_HARDDISK_VBe2585d74-5267121b -f
-    mkfs.lustre --reformat --failover 10.73.20.22@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=0 --mgsnode=10.73.20.11@tcp oss1/ost00
+    mkfs.lustre --reformat --failover oss2.lfs.local@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=0 --mgsnode=mds1.lfs.local@tcp oss1/ost00
     zfs compression=on oss1
     mkdir -p /lustre/zfsmo/ost00
     mount -t lustre oss1/ost00 /lustre/zfsmo/ost00
 
     vagrant ssh oss2
     sudo -s
     zpool create oss2 -o cachefile=none raidz2 /dev/disk/by-id/ata-VBOX_HARDDISK_VBb7776d10-aba16176 /dev/disk/by-id/ata-VBOX_HARDDISK_VB6657241f-bab2ffed  /dev/disk/by-id/ata-VBOX_HARDDISK_VB0ebfc209-9bbf80c4 /dev/disk/by-id/ata-VBOX_HARDDISK_VB51449aa2-95df9cdc -f
-    mkfs.lustre --reformat --failover 10.73.20.21@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=1 --mgsnode=10.73.20.11 oss2/ost01
+    mkfs.lustre --reformat --failover oss1.lfs.local@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=1 --mgsnode=mds1.lfs.local oss2/ost01
     zfs compression=on oss2
     mkdir -p /lustre/zfsmo/ost01
     mount -t lustre oss2/ost01 /lustre/zfsmo/ost01
 ```
 
-After all the commands for each had run successfully, use the IML GUI to scan for the filesystem:
+After all the commands for each node had run successfully, use the IML GUI to scan for the filesystem:
     Configuration -> Servers -> Scan for Filesystem.  Use all servers
 
-If Succussfull, in the IML GUI, the filesystem will be available.
+If Successful, in the IML GUI, the filesystem will be available.
 
 ## Setting up Clients
-In your vagrant folder, run the following script to prepare both client c1 and c2:
-```
+In your vagrant folder, run the following script to prepare both client c1 or c2:
+```
+vagrant sh c1 -c '\
+sudo yum -y install epel-release && \
+sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo && \
+sudo yum-config-manager --add-repo http://mirror.centos.org/centos/7/extras/x86_64/ && \
```

Comment:
The epel and extras repos are not necessary on clients and the client repo should be changed to https://build.whamcloud.com/lustre-b2_10_last_successful_client/.